packages/powersync_attachments_helper/example/getting_started.dart

```diff
@@ -1,7 +1,7 @@
 import 'dart:async';
 
 import 'package:powersync_attachments_helper/powersync_attachments_helper.dart';
-import 'package:powersync/powersync.dart';
+import 'package:powersync_core/powersync_core.dart';
```

Comment:
Is this the recommended approach for this package? Would it not be imported from the `powersync` or SQLCipher version package?

Reply:
For a typical app you'd use one of those two. If you need to be compatible with both (like this helper lib), you'd use `powersync_core`.

Reply:
I can add a comment showing that you can use `powersync` or `powersync_sqlcipher` in a client app?

Reply:
I see. I didn't scan through the entire file. I thought this example snippet actually instantiated a PowerSync client. I see it's just importing the type from the core package.